libpod/sql_state_internal.go

```diff
@@ -384,6 +386,19 @@ func ctrFromScannable(row scannable, runtime *Runtime, specsDir string) (*Contai
 	ctr.valid = true
 	ctr.runtime = runtime
 
+	// Ensure the lockfile exists
```

Comment:
Why do we need to ensure the lockfile exists?  If it does not, thenstorage.GetLockfile will create it?

Reply:
If the lockfile doesn't exist, that means something strange has happened, as the container should have made it when it first started. But even so this is probably a pointless check. Removing.